source/includes/_188-config-repos.md.erb

```diff
@@ -1,5 +1,5 @@
-# Config Repository
+# Pipeline as Code
```

Comment:
I am not sure if calling this `Pipeline as Code` is ok, I think it should still be the name of the resource. Probably the description can have details bout the intent of this API.

I know this can be a little confusing as we already have a `Configuration Repository` API. Lets see what others think.